cmd/kogito/command/shared/resource_checks.go

```diff
@@ -83,7 +83,8 @@ func checkKogitoOperatorExists(kubeCli *client.Client, namespace string) (bool,
 	}
 
 	if operatorDeployment.Status.AvailableReplicas == 0 {
-		return true, fmt.Errorf("Kogito Operator seems to be created in the namespace '%s', but there's no available pods replicas deployed ", namespace)
+		log.Warnf("Kogito Operator seems to be created in the namespace '%s' but there's no available pods replicas deployed ", namespace)
+		return true, nil
```

Comment:
Why did you change this ?
How can we distinguish an operator existing and running to one which is not existing ?
Or should we provide a new method `checkKogitoOperatorRunning` that checks the number of available replica as well (and we could remove this part) ?